python/acquire/__init__.py

```diff
@@ -188,9 +192,13 @@ def next_frame() -> Optional[npt.NDArray[Any]]:
                 if packet := runtime.get_available_data(stream_id):
                     n = packet.get_frame_count()
                     nframes[stream_id] += n
-                    logging.info(f"[stream {stream_id}] frame count: {nframes}")
+                    logging.info(
+                        f"[stream {stream_id}] frame count: {nframes}"
+                    )
                     f = next(packet.frames())
-                    logging.debug(f"stream {stream_id} frame {f.metadata().frame_id}")
+                    logging.debug(
+                        f"stream {stream_id} frame {f.metadata().frame_id}"
```

Comment:
In general, I avoid f-strings for logging (and in particular debug logs) because the f-string is evaluated even if the log is not. Instead I use the varargs/printf style that is baked into logging.

```suggestion
                        "stream %d frame %d", stream_id, f.metadata().frame_id
```

Not specific to this PR, but likely want to make this change at somepoint.